src/manifest.c

```diff
@@ -554,6 +564,13 @@ void subtract_manifests(struct manifest *m1, struct manifest *m2)
 		}
 		list2 = g_list_next(list2);
 	}
+
+	/* now recurse into the included manifests and subtract those as well */
+	while (includes) {
+		mi = includes->data;
+		includes = g_list_next(includes);
+		subtract_manifests(m1, mi, includes);
```

Comment:
I'm also not seeing all included bundles being subtracted.

For example, "devtools-extras" includes "R-extras", which includes "R-basic". However, I'm only seeing "R-extras" subtracted.